components/AlertNotification/Alerts.enum.ts

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+export enum Variant {
```

Comment:
Rename to AlertVariants. Variant is a very common name and will be mistaken for other things. This comes in handy when you use IDE code hinting.